File: mod_02_logica_e_algoritmos/imc/python_imc_oo/TesteImc.py
from PessoaImc import PessoaImc

# if __name__ == "__main__":
#     pessoa = PessoaImc("Fulano", 18, 70, 1.72)
#     print("IMC: ", pessoa.getImc())
#     print("Classificação: ", pessoa.classificacao())


from PessoaImc import PessoaImc

if __name__ == "__main__":
    nome = input("Digite o nome: ")
    idade = int(input("Digite a idade: "))
    peso = float(input("Digite o peso (em kg): "))
    altura = float(input("Digite a altura (em metros): "))

    pessoa = PessoaImc(nome, idade, peso, altura)

    print("IMC: ", pessoa.getImc())
    print("Classificação: ", pessoa.classificacao())

File: mod_02_logica_e_algoritmos/imc/cpp_imc_oo/TesteImc.cpp
#include <iostream>
#include <string>
#include "PessoaImc.h"

int main() {
    std::string nome;
    int idade;
    double peso, altura;

    std::cout << "Digite o nome: ";
    std::getline(std::cin, nome);

    std::cout << "Digite a idade: ";
    std::cin >> idade;

    std::cout << "Digite o peso (em kg): ";
    std::cin >> peso;

    std::cout << "Digite a altura (em metros): ";
    std::cin >> altura;

    PessoaImc pessoa(nome, idade, peso, altura);

    std::cout << "IMC: " << pessoa.getImc() << std::endl;
    std::cout << "Classificação: " << pessoa.classificacao() << std::endl;

    return 0;
}
